frontend/components/Editor.js

```diff
@@ -903,6 +907,9 @@ patch: ${JSON.stringify(
                 }
                 e.preventDefault()
             } else if (e.key.toLowerCase() === "s" && has_ctrl_or_cmd_pressed(e)) {
+                // If VSCode is around, we shouldn't 'Set and run all remote changed remote cells.
+                // Control + Save sends VSCode save
+                if (vscode_available) return this.run_selected()
```

Comment:
I don't understand this change, could you write a more elaborate comment @pankgeorg ?

Reply:
Maybe give a 1.2.3. timeline of what happens when you press Ctrl+S?

Reply:
I think the `Ctrl+S` is pretty much the central TODO that's keeping us from releasing the extension publicly, once we figure this out it's all beautiful 🌟

Reply:
I reverted the change and I'm going all in to server-aware un-run changes. wip wip wip wip